### PR TITLE
Hide search from homepage main when using new topbar

### PR DESCRIPTION
**Pull request by Gnito**
Short paddings:
screen shot 2016-08-04 at 01 33 16
vs
expanded padding-top:
screen shot 2016-08-04 at 01 31 36

The latter takes same height as narrow cover image in other pages:
screen shot 2016-08-04 at 01 32 34

### CODE REVIEW

File: app/views/homepage/index.haml [hunk 1]
```diff
@@ -15,20 +15,22 @@
...
     %p= community_description.html_safe
-    .search-form
-      - if @current_community.private?
-        %a.inline-big-button{href: sign_up_path}
-          .content
-            = t("layouts.application.connect")
-      - else
-        - if(location_search_in_use)
-          = render partial: "location_bar"
+    - if !(is_homepage && !@current_community.private? && FeatureFlagHelper.feature_enabled?(:topbar_v1))
```
Comment (lyyder) [inline app/views/homepage/index.haml]: Could `unless` be used instead of `if !`?

Comment (Gnito) [inline app/views/homepage/index.haml]: Yes, but I rather keep if - else structures in if mode.

Comment (lyyder) [inline app/views/homepage/index.haml]: But does this `if` have a matching `else`?

Comment (Gnito) [inline app/views/homepage/index.haml]: Ah, good point - I was a bit blind here.

**Pull request merged**